kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Environment.java

```diff
@@ -30,10 +30,14 @@ private Environment() {
     public static final String STRIMZI_FEATURE_GATES_ENV = "STRIMZI_FEATURE_GATES";
     private static final String CONTAINER_CONFIG_PATH_ENV = "CONTAINER_CONFIG_PATH";
     private static final String VAULT_CHART_VERSION_ENV = "VAULT_CHART_VERSION";
+    private static final String AWS_LOCALSTACK_CHART_VERSION_ENV = "AWS_LOCALSTACK_CHART_VERSION";
     private static final String SKIP_STRIMZI_INSTALL_ENV = "SKIP_STRIMZI_INSTALL";
     private static final String KAFKA_CLIENT_ENV = "KAFKA_CLIENT";
     private static final String STRIMZI_VERSION_ENV = "STRIMZI_VERSION";
     private static final String CLUSTER_DUMP_DIR_ENV = "CLUSTER_DUMP_DIR";
+    private static final String AWS_ACCESS_KEY_ID_ENV = "AWS_ACCESS_KEY_ID";
+    private static final String AWS_SECRET_ACCESS_KEY_ENV = "AWS_SECRET_ACCESS_KEY";
+    private static final String AWS_USE_CLOUD_ENV = "AWS_USE_CLOUD";
```

Comment:
How about just 
```suggestion
    private static final String USE_CLOUD_ENV = "USE_CLOUD";
```
and we follow up with a separate PR to allow the vault tests to switch between on cluster and hosted based on the same flag? Or maybe it should be `USE_HOSTED_KMS` as `USE_CLOUD` might be a bit too generic...

thoughts: @franvila? @k-wall?

Reply:
Vault is not ready to be used on Cloud in the system tests, the only one currently supported is AWS. I would avoid confusion to the user running the tests. I'd keep `AWS_USE_CLOUD`

Reply:
Long term @SamBarker  is right, we will want to system test Vault against both Vault HCP and on-prem.   But I think @franvila approach is good enough for now.